legend-engine-authentication-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/auth/AuthenticationMethod.java

```diff
@@ -0,0 +1,47 @@
+package org.finos.legend.engine.protocol.pure.v1.model.packageableElement.auth;
```

Comment:
Do we need it in protocol module? Is it runtime concepts?

Reply:
Moved all the runtime stuff including this class to legend-engine-authentication-executionPlan